Polyhedron/demo/Polyhedron/Plugins/IO/OFF_io_plugin.cpp

```diff
@@ -164,12 +165,15 @@ Polyhedron_demo_off_plugin::load_off(QFileInfo fileinfo) {
   Scene_surface_mesh_item* item = new Scene_surface_mesh_item(surface_mesh);
   item->setName(fileinfo.completeBaseName());
   std::size_t isolated_v = 0;
+  std::size_t nm_v = 0;
   for(vertex_descriptor v : vertices(*surface_mesh))
   {
     if(surface_mesh->is_isolated(v))
     {
       ++isolated_v;
     }
+    if(CGAL::Polygon_mesh_processing::is_non_manifold_vertex(v, *surface_mesh))
```

Comment:
use `non_manifold_vertices()` that takes an output iterator as paramater.

Reply:
No. `non_manifold_vertices()` stores one halfedge per umbrella incident to the vertex, so you have no way to know the actual number of nm_vertices. You may have 4 umbrellas incident to vertex A, 2 to vertex B and 2 to vertex C, or 6 to vertex A and 2 to vertex B, per example.

Reply:
so what? even a post processing using a `std::set` will be faster than something that is O(n^2).